### PR TITLE
Qualcomm family override

### PR DESCRIPTION
**Pull request by lumag**

### CODE REVIEW

File: conf/machine/include/qcom-apq8064.inc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-SOC_FAMILY = "apq8064"
+SOC_FAMILY = "apq8064:qcom"
```
Comment (ndechesne) [inline conf/machine/include/qcom-apq8064.inc:1]: hmm. shouldn't it be qcom:apq8064, e.g. from more generic to less generic?

File: conf/machine/include/qcom-apq8016.inc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-SOC_FAMILY = "apq8016"
+SOC_FAMILY = "apq8016:qcom"
```
Comment (ndechesne) [inline conf/machine/include/qcom-apq8016.inc:1]: perhaps we can create include/qcom.inc with:

SOC_FAMILY = "qcom"
require conf/machine/include/soc-family.inc

**Comment by ndechesne** [comment]
it looks much better with a single override!

File: conf/machine/include/qcom-apq8016.inc (hunk 2 above)
Comment (lumag) [reply conf/machine/include/qcom-apq8016.inc:1]: I'd probably pick up a patch from #147

Comment (ndechesne) [reply conf/machine/include/qcom-apq8016.inc:1]: makes sense. yes.

File: conf/machine/include/qcom-apq8064.inc (hunk 1 above)
Comment (lumag) [reply conf/machine/include/qcom-apq8064.inc]: done, in `qcom-common.inc`

File: conf/machine/include/qcom-apq8016.inc (hunk 2 above)
Comment (lumag) [reply conf/machine/include/qcom-apq8016.inc]: done

**Pull request merged**